src/lib/check-flags.js

```diff
@@ -0,0 +1,13 @@
+export default (item, newItem) => {
+    // check flags
+    const itemFlags = item.flags
+    const newItemFlags = newItem.flags
+    if (!itemFlags && !newItemFlags) {
+      return true
+    } else if (!itemFlags && newItemFlags || (itemFlags && !newItemFlags)) {
+      return false
+    } else {
+      return itemFlags.every(flag => newItemFlags.includes(flag))

```

Comment:
Dá pra passar, mas (FYI) isso não vai funcionar se `itemFlags` for um array vazio ou se tiver menos flags que o do novo item

Reply:
E cagou indentação aqui, o que me leva a crer que não tá usando o linter 😞 , nem testando...